static/src/javascripts/bootstraps/commercial.js

```diff
@@ -28,24 +28,24 @@ import { initCheckDispatcher } from 'commercial/modules/check-dispatcher';
 import commentAdverts from 'commercial/modules/comment-adverts';
 
 const commercialModules: Array<Array<any>> = [
-    ['cm-highMerch', initHighMerch],
     ['cm-thirdPartyTags', initThirdPartyTags],
     ['cm-prepare-googletag', prepareGoogletag, true],
     ['cm-closeDisabledSlots', closeDisabledSlots],
-    ['cm-paidContainers', paidContainers],
-    ['cm-paidforBand', initPaidForBand],
     ['cm-carrot', initCarrotTrafficDriver],
```

Comment:
If the rest of the `paidFor` content is getting moved into the `if (!commercialFeatures.adFree) {` the cm-carrot could probably go along with them

Not a major issue since IIRC the carrot is currently switched off

Reply:
Thanks Kate. I assumed the carrot thing was an acquisitions device, and wouldn't pester paying users. I'll add a card to investigate this further.